freva-rest/src/tool_api/endpoints.py

```diff
@@ -0,0 +1,45 @@
+from typing import Dict
```

Comment:
what is this endpoints and the other in `freva-rest/src/freva_rest/tool_api/endpoints.py`? could u plz concatenate them in one?

Reply:
You are right, I removed the superfluous folder now